solutions/devsprint-eduardo-bocato-1/SpaceApp/Modules/Core/DesignSystem/DS+Colors.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  DS+Colors.swift
+//  SpaceApp
+//
+//  Created by Pedro Gabriel on 14/03/22.
+//
+
+import SwiftUI
+
+extension DS {
+    public struct Color {
+        let color: SwiftUI.Color
+    }
+
+    public struct DynamicColor {
+        @Environment(\.colorScheme) private var colorScheme
```

Comment:
**[CLARIFICATION]**
Do anyone needs further explanation about how this works?

Reply:
For me, it's ok.